src/amo/components/Link.js

```diff
@@ -12,33 +12,36 @@ export class LinkBase extends React.Component {
     base: PropTypes.string,
     children: PropTypes.node,
     href: PropTypes.string,
+    prefix: PropTypes.bool,
     to: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
   }
 
   static defaultProps = {
     base: '',
+    prefix: true,
   }
 
   render() {
-    const { base, children, href, to, ...customProps } = this.props;
+    const { base, children, href, prefix, to, ...customProps } = this.props;
 
     if (typeof href === 'string' && typeof to !== 'undefined') {
       throw new Error(
         'Cannot use "href" prop and "to" prop in the same Link component');
     }
 
     if (typeof href === 'string') {
-      const linkHref = href.startsWith('/') ? path.join(base, href) : href;
+      const linkHref = href.startsWith('/') && prefix ?
+        path.join(base, href) : href;
       return <a {...customProps} href={linkHref}>{children}</a>;
     }
 
     let linkTo = to;
-    if (typeof to === 'string' && to.startsWith('/')) {
-      linkTo = path.join(base, to);
+    if (typeof to === 'string') {
+      linkTo = to.startsWith('/') && prefix ? path.join(base, to) : to;
```

Comment:
That isn't an intent, no. The `to` attribute is really only for local links and I'm not in love with this behaviour but I'm just emulating what the original Link component does.

Maybe what we should really be doing is throwing an error about relative URLs because really we should never be using them.